client/ayon_deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -13,7 +13,7 @@
     FileUtils,
     DirectoryUtils,
 )
-__version__ = "1.1.3"
+__version__ = "1.1.4"
```

Comment:
This deserves minor bump.
```suggestion
__version__ = "1.2.0"
```